src/applications/disability-benefits/4142/config/form.js

```diff
@@ -0,0 +1,252 @@
+// _ from 'lodash/fp';
+
+// Example of an imported schema:
+// import fullSchema from '../22-4142-schema.json';
+// In a real app this would be imported from `vets-json-schema`:
```

Comment:
We'll probably want to import the schema from vets-json-schema (rather than include it here), as noted in this comment.

Reply:
Yep, I'm all for leaving it in vets-website until it settles.